test/declaration_test.ts

```diff
@@ -389,6 +389,13 @@ abstract class X {
   external get y;
   external set y(v);
   external factory X({String x, y});
+}`);
+    expectTranslate(`interface X { '!@#$%^&*': string; }`).to.equal(`@anonymous
+@JS()
+abstract class X {
+  /*external String get !@#$%^&*;*/
```

Comment:
nit: to make the test case better have multiple failing cases and a passing case like
"foo3481$"